src/MutantRunResultMatcher.ts

```diff
@@ -26,12 +26,26 @@ export default class MutantRunResultMatcher {
     });
   }
 
-  private mutantCovers(mutant: Mutant, location: Location): boolean {
+  private mutantCoversFile(mutant: Mutant, coveredFile: CoverageResult): boolean {
+    let smallestStatementFound = "";
+    
+    Object.keys(coveredFile.statementMap).forEach(statementId => {
+      let location = coveredFile.statementMap[parseInt(statementId)];
+        
+      if(this.mutantCoversLocation(mutant, location)){
+        smallestStatementFound = statementId;
```

Comment:
This implementation is actually very much dependent on the order in which the statement map is build. I think this is too error prone. What if Istanbul decides to change the order? I think it would be better to compare the location of the statements that cover the mutant.

if nr of lines of the location is smaller OR (the nr of lines are the same AND the number of columns is smaller)... something like that